server/tests/utils/test_error.py

```diff
@@ -0,0 +1,33 @@
+"""Test the Error util module."""
```

Comment:
no real reason to test AppError since it just holds two variables and spits them out later, so just tested the decorator.